google-http-client/src/main/java/com/google/api/client/http/GenericUrl.java

```diff
@@ -503,6 +561,20 @@ public void appendRawPath(String encodedPath) {
    *     or {@code ""} input
    */
   public static List<String> toPathParts(String encodedPath) {
+    return toPathParts(encodedPath, false);
+  }
+
+  /**
+   * Returns the decoded path parts for the given encoded path.
+   *
+   * @param encodedPath slash-prefixed encoded path, for example {@code
+   *     "/m8/feeds/contacts/default/full"}
+   * @param verbatim flag the specifies whether parts should be decoded or not.
+   * @return decoded path parts, with each part assumed to be preceded by a {@code '/'}, for example
```

Comment:
not necessarily decoded

Reply:
Done

Reply:
I don't see any change. I suggest

"@return path parts (decoded if not {@code verbatim}), with each part ..."